bundles/EcommerceFrameworkBundle/VoucherService/TokenManager/Single.php

```diff
@@ -134,9 +137,13 @@ public function insertOrUpdateVoucherSeries()
                     ON DUPLICATE KEY UPDATE token = ?, length = ?';
 
             $db->query($query, [trim($this->configuration->getToken()), $this->getFinalTokenLength(), $this->getSeriesId(), trim($this->configuration->getToken()), $this->getFinalTokenLength()]);
+
+            return true;
         } catch (\Exception $e) {
-            return ['error' => 'Something went wrong.'];
+            return false;
```

Comment:
I think we at least should log the exception...

Reply:
Well i agree with you - however there is absolutely nothing logged within the whole VoucherService folder. There is a bunch commented log messages here and there and a TODO message: "// TODO - Log Errors".

IMHO the whole 'VoucherService' could use a little polish. I think we should cover that in a seperate issue.

What do you think @fashxp ?